packages/commonwealth/client/scripts/controllers/server/threads.ts

```diff
@@ -389,7 +389,7 @@ class ThreadsController {
         if (result.stage === ThreadStage.Voting) this.numVotingThreads++;
         // Post edits propagate to all thread stores
         this._store.update(result);
-        this._listingStore.update(result);
+        this._listingStore.add(result);
```

Comment:
Why did these need to get changed?

Reply:
this was the root of the refresh problem- the add method needed to be used to correctly update the listing store so it would correctly set the posts without a refresh